src/App/index.jsx

```diff
@@ -1,6 +1,24 @@
+import { parse } from 'query-string';
+import { Redirect, Route, Switch } from 'react-router-dom';
 import Benchmark from '../views/Benchmark';
 import './metricsGraphics.css';
 
-const App = () => <Benchmark />;
-
-export default App;
+export default () => (
+  <Switch>
+    <Route
+      path="/"
+      render={({ location }) => {
+        const { platform, benchmark } = parse(location.search);
+        if (!platform || !benchmark) {
+          return <Redirect to="/?platform=win10&benchmark=motionmark-animometer" />;
+        }
```

Comment:
You could just define these defaults inside the view/component and use them if nothing is passed in. This would remove the need for the redirect.